community/asset_library/submitting_to_assetlib.rst

```diff
@@ -178,6 +178,11 @@ is required in the submission form here as well.
     The **icon** must be square (1:1 aspect ratio). It should have a minimum
     resolution of 128×128 pixels.
 
+.. note::
+
+    If you are hosting your repository with GitHub you can host the icon within the repo.
+    You must provide a url in the form `https://raw.githubusercontent.com/<user>/<project>/<branch>/Icon.png`.
```

Comment:
```suggestion
    For icons hosted on GitHub, you must provide a URL in the form of `https://raw.githubusercontent.com/<user>/<project>/<branch>/Icon.png`.
```

This gets closer to mentioned error message, is a bit more consistent with the docs style wise, and does only inform, not recommend using Github for hosting (and I'm unsure whether using it for webhosting is within Github's TOS).